src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -21,6 +21,6 @@ public static int getFibonacciNumber(int index) {
         if (index < 2) {
             return 1;
         }
-        return getFibonacciNumber(index - 1) + getFibonacciNumber(index - 2);
+        return getFibonacciNumber(--index) + getFibonacciNumber(index - 1);
```

Comment:
yes, I think about it
and we can write:
return getFibonacciNumber(index - 2) + getFibonacciNumber(--index);